src/main/include/NetworkTableUtil.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 FRC Team 3512. All Rights Reserved.
+
+#pragma once
+
+#include <initializer_list>
+#include <type_traits>
+
+#include <networktables/NetworkTableEntry.h>
+#include <networktables/NetworkTableInstance.h>
+#include <wpi/Twine.h>
+
+namespace frc3512::NetworkTableUtil {
+
+/**
+ * Creates a networktable entry that pushes a default value to the
+ * networktable. Works for most types.
+ *
+ * @param name Key
+ * @param defaultValue of the type that the entry will be
```

Comment:
Run `./gradlew doxygen`, then open `docs/html/index.html` to make sure this documentation turned out right. I think this will produce an incomplete sentence.